aoa_counter/localdata/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+    "camera": {
+        "protocol": "http",
+        "ip": "127.0.0.1",
+        "port": 80,
+        "user": "root",
+        "pass": "",
+        "serviceID": 1,
+        "fieldName": ""
```

Comment:
tady by mohlo byt mozna defaultne field1 - mrkni jak se to jmenuje v CO kdyz vytvoris pole